### PR TITLE
Make the staging/"watch" version of the documentation use the local `handsontable` and wrapper builds.

### PR DESCRIPTION
**Pull request by jansiegel**
### Context
This PR:
- Makes the docs symlink the wrappers alongside `handsontable`
- Make the `next`-based docs use the locally symlinked versions of the wrappers
- Make the staging workflow build `handsontable` + the wrappers before building the docs to use the current state of the repo

_[skip changelog]_

### How has this been tested?
- Manually by running `npm run start:no-cache`
- [By committing a broken react wrapper and checking if staging environment fails](https://github.com/handsontable/handsontable/actions/runs/4617671232/jobs/8164194825)

### Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature or improvement (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)
- [ ] Additional language file or change to the existing one (translations)

### Related issue(s):
1. handsontable/dev-handsontable#1232

### Affected project(s):
- [ ] `handsontable`
- [ ] `@handsontable/angular`
- [ ] `@handsontable/react`
- [ ] `@handsontable/vue`
- [ ] `@handsontable/vue3`

### Checklist:

- [x] I have reviewed the guidelines about [Contributing to Handsontable](https://github.com/handsontable/handsontable/blob/master/CONTRIBUTING.md) and I confirm that my code follows the code style of this project.
- [ ] I have signed the [Contributor License Agreement](https://docs.google.com/forms/d/e/1FAIpQLScpMq4swMelvw3-onxC8Jl29m0fVp5hpf7d1yQVklqVjGjWGA/viewform?c=0&w=1)
- [ ] My change requires a change to the documentation.

### CODE REVIEW

**Review by budnix — CHANGES_REQUESTED** [review]
I've deployed the changes to staging, for example, https://dev.handsontable.com/docs/react-data-grid/binding-to-data/. The wrapper dist files return 404.

**Comment by jansiegel** [comment]
> I've deployed the changes to staging, for example, https://dev.handsontable.com/docs/react-data-grid/binding-to-data/. The wrapper dist files return 404.

@budnix 
Should work fine after https://github.com/handsontable/handsontable/pull/10319/commits/58523728efef5dd2597b0e45dfa3e083c082616b

File: docs/.vuepress/handsontable-manager/dependencies.js [hunk 1]
```diff
@@ -2,23 +2,52 @@
...
+ * @returns {string} Url to the required file.
+ */
+const getPackageUrls = (packageName, version, fileSelection) => {
+  const subDirs = {
+    handsontable: {
+      js: 'handsontable.full.min.js',
+      css: 'handsontable.full.min.css'
+    },
+    '@handsontable/react': {
+      js: 'react-handsontable.js'
```
Comment (budnix) [inline docs/.vuepress/handsontable-manager/dependencies.js:22]: Why here is the non-minified version used when all others use min? 😄

**Review by budnix — APPROVED** [review]

File: docs/.vuepress/handsontable-manager/dependencies.js (hunk 1 above)
Comment (jansiegel) [reply docs/.vuepress/handsontable-manager/dependencies.js]: Good point

**Review by budnix — APPROVED** [review]

**Pull request merged**